### PR TITLE
Feature filters

### PR DESCRIPTION
**Pull request by OmKakatkar**
# Features

## Additions
* Filters for Priority and Tags
* Sort by Date (latest or oldest first)

### CODE REVIEW

**Review by sanjay-xdr — COMMENTED** [review]
Everything looks good 
few things you can improve 
Card design 
and in Filter functionality when u sort according to date the order of tags are also getting change

File: src/frontend/components/Filter/Filter.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.filter-type-container,
+.filter-list {
+	margin-top: 0.5rem;
+}
+
+.filter-type-container {
+	padding: var(--padding-1r);
```
Comment (mimansha-swarup) [inline src/frontend/components/Filter/Filter.css:7]: great practice

File: src/frontend/components/NoteUpdateModal/NoteUpdateModal.js [hunk 2]
```diff
@@ -11,15 +11,17 @@ import { RichTextEditor } from "../RichTextEditor/RichTextEditor";
...
-		priority: priority,
-		tags: tags,
-		_id: _id,
+		title,
+		description,
+		noteColor,
+		priority,
+		tags,
+		createdAt,
+		_id,
```
Comment (mimansha-swarup) [inline src/frontend/components/NoteUpdateModal/NoteUpdateModal.js:24]: again another good practice

File: src/frontend/pages/Notes/Notes.jsx [hunk 3]
```diff
@@ -15,10 +15,13 @@ import { useAuth } from "../../context/auth-context";
 import { useNotes } from "../../context/notes-context";
 import { addToArchives } from "../../services/notes-archive-service";
 import { addToTrash } from "../../services/notes-trash-service";
```
Comment (mimansha-swarup) [inline src/frontend/pages/Notes/Notes.jsx:17]: suggestion: barrel export can be used

**Review by mimansha-swarup — APPROVED** [review]
very clean and modular code 
easy to read

File: src/frontend/pages/Notes/Notes.jsx (hunk 3 above)
Comment (OmKakatkar) [reply src/frontend/pages/Notes/Notes.jsx:17]: Thanks for the suggestion, this will be done in a future PR with refactored code

**Pull request merged**